src/ucp/core/ucp_worker.h

```diff
@@ -194,6 +194,7 @@ typedef struct ucp_worker {
     unsigned                      ep_config_count;/* Current number of configurations */
     ucp_ep_config_t               ep_config[0];  /* Array of transport limits and thresholds */
     ucs_cpu_set_t                 cpu_mask;      /* Save CPU mask for subsequent calls to ucp_worker_listen */
+    uint64_t                      am_message_id;
```

Comment:
this is only for tags, pls put inside "tm" context as "message_id" and initialize it as part of tm context

Reply:
updated